cmake/FindZ3.cmake

```diff
@@ -67,15 +67,15 @@ if (NOT Z3_FOUND)
 	unset(Z3_VERSION_STRING)
 
 	# Try to check version by compiling a small program that prints Z3's version
-	if(Z3_CXX_INCLUDE_DIRS AND Z3_LIBRARIES)
+	if (Z3_CXX_INCLUDE_DIRS AND Z3_LIBRARIES)
 		check_z3_version(${Z3_CXX_INCLUDE_DIRS} ${Z3_LIBRARIES})
-	endif()
+	endif ()
 
-	if(NOT Z3_VERSION_STRING)
+	if (NOT Z3_VERSION_STRING)
 		set(Z3_VERSION_STRING "0.0.0")
-	endif()
+	endif ()
 
-	include (FindPackageHandleStandardArgs)
+	include(FindPackageHandleStandardArgs)
 	find_package_handle_standard_args(Z3
```

Comment:
why the whitespace changes throughout this file?

But, more importantly, this file should not be needed at all anymore, since this is all supposed to be handled by the LogicBlocks module.